junifer/markers/utils.py

```diff
@@ -5,15 +5,17 @@
 #          Sami Hamdan <s.hamdan@fz-juelich.de>
 #          Synchon Mandal <s.mandal@fz-juelich.de>
 #          Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Amir Omidvarnia <a.omidvarnia@fz-juelich.de>
 # License: AGPL
 
 from typing import Any, Callable, Dict, List, Type, Union
 
+import neurokit2 as nk
```

Comment:
Good to have a `try...catch` here.

Reply:
Sure, we will do it with Fede.

Reply:
With the refactored versions, this is not relevant anymore.